samples/spring-client-java-valueentity-counter/src/main/proto/com/example/counter_api.proto

```diff
@@ -0,0 +1,47 @@
+// This is the public API offered by your entity.
+syntax = "proto3";
+
+package com.example; // <1>
+
+import "akkaserverless/annotations.proto"; // <2>
```

Comment:
same as above

Reply:
Removed akkaserverless annotations from all the proto files.